website/docs/sdk-reference/python.md

```diff
@@ -266,9 +266,9 @@ client = configcatclient.get('#YOUR-SDK-KEY#',
 
 Available options:
 
-| Option Parameter             | Description                  | Default |
-| ---------------------------- | ---------------------------- | ------- |
-| `cache_time_to_live_seconds` | Cache TTL.                   | 60      |
+| Option Parameter                 | Description                  | Default |
+| -------------------------------- | ---------------------------- | ------- |
+| `cache_refresh_interval_seconds` | Cache TTL.                   | 60      |
```

Comment:
Is this description enough to understand the concept of the parameter?

Reply:
I think yes. This PR is just bringing back the removed changes that have already been approved/merged earlier.
If you would like to improve this please open a new PR with your suggestion 👍 .